### DISCUSSION TITLE
Is there a way to config maxLenght on dynamic mask?

### DISCUSSION THREAD

**Discussion by AlvaroHeringer** [uNmAnNeR/imaskjs#812] [Q&A]
I'm trying to create an input that accepts a phone or email. When I exceed the amount of phone characters, the input loses the phone mask and I would like to add a max to it. And I want the email mask without character limits.

I'm currently using this template:
```
{
  mask: '+{7}(000)000-00-00',
 },
{
  mask: /^\S*@?\S*$/
}

```

I tryed to add 'maxLenght: 11' on phone object but it doesn't work.

Thank you!

**Comment by uNmAnNeR** [comment]
you have to make sure that first part of your email is not satisfies condition for phone number. now phone number satisfies `\S*` so it's correct behaviour
